core/test/lib/lantern/page-dependency-graph-test.js

```diff
@@ -0,0 +1,623 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
```

Comment:
```suggestion
 * Copyright 2024 Google LLC
```

Reply:
I'm not sure what the rules are, but the file this is testing is marked `2024`. Up to you but should at least be consistent.